### ISSUE TITLE
error: unable to open output file 'lib/libc4.a': 'No such file or directory'

### ISSUE THREAD

**Issue by VinayIsHere** [Avalanche-io/libc4#1]
I just clone your ```repo``` and install the both library dependency but when I run the ```make``` command it shows me the below error.
```
clang -o lib/libc4.a lib.c -I/usr/local/Cellar/openssl/1.0.2l/include -c
error: unable to open output file 'lib/libc4.a': 'No such file or directory'
1 error generated.
make: *** [Makefile:10: lib/libc4.a] Error 1

```

**Comment by mrjoshuak** [comment]
That's caused because the lib folder doesn't exist, git does not track empty folders. I'll add a lib folder dependency to the to the Makefile.  As a workaround for now just create the `lib` directory before you run make.

**Comment by mrjoshuak** [comment]
addressed in c674678910ba46ccb413141c6c3abd5dba054ea2
